app/lib/project/MetapolatorProject.js

```diff
@@ -614,6 +616,38 @@ define([
         exportController.export();
     };
 
+    _p.exportZipInstance = function(masterName, instanceName, precision) {
+        //export the font to a temporary directory
+        var temp_dir = instanceName+"_temp";
+        this.exportInstance(masterName, temp_dir, precision);
+
+        //encode it as a zip file
+        var content = zip.encode(this._io, temp_dir);
+        this._io.writeFile(false, instanceName, new Buffer(content, 'base64'));
+
+        //delete the temp dir
+        function rmDirRecursive(io, dir){
+            function _isDirName(name) {
+                return name.slice(-1) === '/';
+            }
+
+            var i;
+            var entries = io.readDir(false, dir);
+            for (i in entries){
```

Comment:
Don't iterate over array this way. In the future of js `for(item of array)` is planned, but we can't use it right now.

you can use `array.prototype.map` or you do
```
  var i;
  for(i=0;i<entries.length;i++)`
// a bit more efficient:
var i,l;
for(i=0, l=entries.length;i<l;i++)
```